package.json

```diff
@@ -10,7 +10,7 @@
     "watch-components": "lerna run watch --parallel",
     "setup-npm-links": "node scripts/setup-npm-link.js",
     "postinstall": "lerna bootstrap --hoist && cross-env NODE_ENV=production npm run build-components && lerna run --parallel genPackagelock",
-    "release": "npm whoami && npx lerna clean && rm -rf node_modules && npm install && npm run test && lerna version --exact && cross-env NODE_ENV=production npm run build-components && lerna publish from-package",
+    "release": "npm whoami && npx lerna clean && rm -rf node_modules && npm ci && npm run test && lerna version --exact && cross-env NODE_ENV=production npm run build-components && lerna publish from-package",
```

Comment:
As long as the `rm -rf` command stays there, i don't see the point in running npm ci. Because the whole purpose seems to be that it automatically does that for you.

And what I'm not sure about: It ALWAYS expects a package-lock file. Is that a good idea? What if you want to update all minor versions at once? By thinking about it, that is probably not a big concern, as npm install can still be run manually.

I would approve if you remove the `rm -rf node_modules`

Reply:
I had to read up on things here: https://medium.com/better-programming/npm-ci-vs-npm-install-which-should-you-use-in-your-node-js-projects-51e07cb71e26 

@danyball A brief explainer would have been helpful.

"
- `npm ci` will delete your node_modules folder to ensure a clean state.
- It will look in your package-lock.json to install all the dependencies with the exact version.
- Unlike npm install, `npm ci` will never modify your package-lock.json. It does however expect a package-lock.json file in your project — if you do not have this file, npm ci will not work and you have to use npm install instead.

If you use npm ci, you’ll get reliable builds."

Convincing to me, and since it is restricted to `release`, OK. We can still update package-lock.json separately, using `npm i`.

Reply:
@markus-walther i have given exact your explainer at my description? 

Jep, will remove that.